components/concept-component/concept-component.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "./concept-component.module.scss";
+
+const ConceptComponent = () => {
+  return (
+    <div className={styles["concept-grid"]}>
+      <div className={styles["concept-wrapp"]}>
+        <h2 className={styles["concept-text"]}>\\ Concepto:</h2>
+      </div>
+      <ul className={styles["search-wrapp"]}>
+        <li className={styles["search-text"]}>Buscar</li>
```

Comment:
This should receive an array of strings, each string will be a `li`

Reply:
@diegoaguilar  ok, so i create the array with somthinh like:
- bullet 1
-bulet 2
... etc, right?